orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/support/SourceResolver.groovy

```diff
@@ -43,8 +44,33 @@ class SourceResolver {
   StageData.Source getSource(Stage stage) throws RetrofitError, JsonParseException, JsonMappingException {
     def stageData = stage.mapTo(StageData)
     if (stageData.source) {
-      // has an existing source, return it
-      return stageData.source
+      // targeting a source in a different account and region
```

Comment:
Looks like we're really just trying to decide whether we're targeting a source cluster? (it may or may not be in a different account?)

If so, can we get away with:

```
if (stageData.source.clusterName && stage.context.target) {
   ...
}
```